bundles/core/src/main/java/com/adobe/cq/wcm/core/components/models/Image.java

```diff
@@ -117,6 +116,12 @@
      */
     String PN_FLIP_VERTICAL = "imageFlipVertical";
 
+    /**
+     * Name of the resource property that defines areas of an image map.
+     *
+     * @since com.adobe.cq.wcm.core.components.models 12.4.0
+     */
+    String PN_MAP = "imageMap";
```

Comment:
It would make more sense to have this as a plural, since it supports multiple maps. We should also document the storage format for this property and this seems like the best place for it.

Reply:
@vladbailescu - this was deliberate to match the HTML markup definition as well as the property name itself. A single `map` has multiple `area`.

Reply:
Got it, thx!

Reply:
Also, I can see the format described at https://github.com/Adobe-Marketing-Cloud/aem-core-wcm-components/pull/181/files#diff-561deebea47b85bc8ffddf78fe11602dR146 but for me it would make more sense to have it in the interface, as it's public.